app/dashboard/templates/profiles/status_box.html

```diff
@@ -29,5 +29,15 @@
       </button>
     </div>
   </div>
+  <div id="thumbnail" class="mt-1 mb-2" style="display: none; top: initial; left: initial; background-color: #FAFAFA">
```

Comment:
Can you add this code also to townsquare/templates/index.html ? now as we changed the design  I had tested and is the only missing change to this work with the current code.
For my side just that and this is ready to get merge. Good work